setup.py

```diff
@@ -9,12 +9,15 @@
 """
 
 from setuptools import setup, find_packages
+import subprocess
 
-version = '0.8.0'
+
+def get_version():
+    return subprocess.check_output(["git", "describe", "--tags"]).strip().decode("utf-8")
 
 setup(
     name = 'multiqc_ngi',
-    version = version,
+    version = get_version(),
     author = 'Phil Ewels',
     author_email = 'phil.ewels@scilifelab.se',
```

Comment:
this email is meaningless, since it is inactive now.

Reply:
Removed author from setup now